packages/ui/components/TextField/TextField.tsx

```diff
@@ -52,12 +52,14 @@ export const BaseTextField: React.FunctionComponent<TextFieldProps> = ({
   const icon2Color = err ? 'error' : iconSecondaryColor;
 
   const states = { loading, suffix, disabled, error };
+  const { name } = props;
+  const excludedIconField = ['email', 'password'];
```

Comment:
Hey @tomoyakuroda! Can you explain the idea behind this?

Reply:
Hello @tristanMatthias, I'm not sure where the input fields are used in the project, so I hide the exclamation mark for only email and password fields so that it doesn't affect other input fields. If we want to remove the exclamation for all input fields, I will change it 😀.

Reply:
Yeah this might cause some strange behavior. If we can, let's remove these for now :). THanks!

Reply:
Sure! I removed these changes.